native/CMakeLists.txt

```diff
@@ -506,10 +506,19 @@ cocos_source_files(
     cocos/platform/UniversalPlatform.h
 )
 if(WINDOWS)
-    cocos_source_files(
-        cocos/platform/win32/WindowsPlatform.cpp
-        cocos/platform/win32/WindowsPlatform.h
-    )
+    if(CC_EDITOR) 
+        cocos_source_files(
+            cocos/platform/editor/windows/EditorWindowsPlatform.cpp
+            cocos/platform/win32/WindowsPlatform.h
+            cocos/platform/editor/windows/Game.cpp
+            cocos/platform/editor/Game.h
```

Comment:
Why do we have to put Editor's `Game.h/.cpp` to engine?

Reply:
I rename it as EditorAppliction.
Put this code to engine is for avoiding compile error when refactor platform codes.